### ISSUE TITLE
FE | Profile | [A11y Profile Audit]: screenreaders, edit direct deposit information, information about where to find account and routing numbers not accessible

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#46614]
### Point of contact

Angela Fowler

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

...
When I choose "Edit direct deposit information," focus is placed in the first edit box. While this is usually best practice, in this case there is important information above the edit box about where to find your account and routing number which a screen reader user may miss. 

That information presents as an image of a check. NVDA reads the paragraph below it detailing how to get the info visually and nonvisuallly. JAWS howeer does not.  

### Reproduction steps

1.
2.
3.
..Go to staging.va.gov, and sign in as user+41.
Choose "Go to my Profile."
In the Profile menu, choose "Direct Deposit Information."
Enable JAWS.
Choose the button: "Edit deposit information..
Focus will be placed in the first edit box.
Arrow up, out of the edit box. JAWS will not see the paragraph detailing how to get the info.

### Proposed solution or next steps

...
Set focus to the h2.

The paragraph with the information is a figure caption. Removing it from the figure element and placing it in a plain paragraph below the image will solve the problem. 

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

Mock:[Here it is](https://www.sketch.com/s/1a920e73-1dcb-47c4-aae8-08656756c131/a/4a38jpJ)

**Comment by adamwhitlock1** [comment]
@andaleliz here is the code to the markup

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/personalization/profile/components/direct-deposit/BankInfo.jsx#L279-L298

**Comment by SKasimow** [comment]
@andaleliz - created #46876 for the creation of a mock

**Comment by andaleliz** [comment]
The mock already exists! [Here it is](https://www.sketch.com/s/1a920e73-1dcb-47c4-aae8-08656756c131/a/4a38jpJ).  I think there was a disconnect between FE and design when the page was originally built.

**Comment by SKasimow** [comment]
Closing - work being done in #47003
